apps/web/utils/auth.ts

```diff
@@ -43,7 +43,7 @@ export const betterAuthConfig = betterAuth({
   },
   baseURL: env.NEXT_PUBLIC_BASE_URL,
   trustedOrigins: [env.NEXT_PUBLIC_BASE_URL],
-  secret: process.env.NEXTAUTH_SECRET,
+  secret: env.BETTER_AUTH_SECRET,
```

Comment:
make this `env.AUTH_SECRET || process.env.NEXTAUTH_SECRET` so it doesn't break the build for everyone self hosting